pkg/acsengine/Get-AzureConstants.py

```diff
@@ -85,6 +85,8 @@ def getFileContents(dcosMasterMap, masterAgentMap, kubernetesAgentMap, sizeMap,
 	"germanynortheast",
 	"usgovvirginia",
 	"usgoviowa",
+	"usgovarizona",
```

Comment:
@jackfrancis , turns out the generator scripts have those regions hardcoded, so I just added them there.